### ISSUE TITLE
Programtically parse wcag2.1 spec

### ISSUE THREAD

**Issue by LukeButters** [w3c/wcag#1729]
Hi, I am trying to programmatically parse the wcag 2.1 spec but I have ran into a number of issues:

1. It is not clear what branch or tag I should be using.
2. FLASH techniques seem to missing from the [techniques folder ](https://github.com/w3c/wcag/tree/main/techniques) yet they are visible on the [w3c site](https://www.w3.org/WAI/WCAG21/Techniques/flash/FLASH15.html)
3. When generating the JSON using the ant json task, some [techniques seem to be missing](https://github.com/w3c/wai-wcag-quickref/issues/204)

Would I be better of just parsing the html documents under https://www.w3.org/WAI/WCAG21/

**Comment by alastc** [comment]
Hi Luke,

I'm not clear what you are trying to get out of the parsing, so I'm not sure what to recommend.

1. There is a WCAG-2.1 branch in github.
2. Flash techniques will go in a future publication-run of the documents.
3. The quickref data is out of date, but if you are parsing github or the public facing HTML pages that wouldn't be the case.

Overall, we don't have capacity to provide support for this kind of request, so I'm going to close the issue.
